app/Importers/DataImporter.php

```diff
@@ -12,16 +12,22 @@
 use App\Models\ImportInfoDetail;
 use App\Services\MapboxGeocodingService;
 use GuzzleHttp\Client;
+use Stichoza\GoogleTranslate\GoogleTranslate;
 
 abstract class DataImporter
 {
+    public const language = "en";
```

Comment:
Const should be UPPERCASE.
(StringManipulation is a good plugin for PHPStorm).

Do you need this constant as `public`? Is it used anywhere else?

Reply:
Yeah, i should make it private, but thanks to this comment I noticed that it should be deleted - DataImporter is not usinh GooggleTranslate anymore